ansible/templates/minigraph_link_meta.j2

```diff
@@ -53,34 +53,23 @@
   </LinkMetadataDeclaration>
 {% endif %}
 
-{% if msft_an_enabled is defined and vm_topo_config.get('autoneg_interfaces') is not none %}
```

Comment:
Recommend a top level Property check here like msft_an_enabled, as we could have a case where we might need to have FecDisabled along with Autoneg property, so  basically we could have different flavors of AutoNeg

Reply:
Hi @vdahiya12 not sure why it's needed "msft_an_enabled". As I understood, we need to disable autoneg when speed is set to 100G? If yes - what about other speed (50G, 400G) for example. Please elaborate more here.

Reply:
@SavchukRomanLv we have a use case which is not as per standard i.e FEC needs to be disabled when AN is enabled and that is the reason for "msft_an_enabled".

Reply:
Maybe a better way is fec_disabled can be entered if msft_an_enabled is defined. 
@SavchukRomanLv lets sync on a meeting about this. Please provide your alias for invite.